docs/sources/flow/reference/components/loki.source.cloudflare.md

```diff
@@ -39,11 +39,11 @@ Name            | Type                 | Description          | Default | Requir
 `workers`       | `int`                | The number of workers to use for parsing logs.     |  `3` | no
 `pull_range`    | `duration`           | The timeframe to fetch for each pull request.      | `"1m"` | no
 `fields_type`   | `string`             | The set of fields to fetch for log entries.        | `"default"` | no
+`additional_fields` | `list(string)`       | The addional list of fields to supplement those provided via `fields_type`. |  | no
 
 
-By default `loki.source.cloudflare` fetches logs with the `default` set of
-fields. Here are the different sets of `fields_type` available for selection,
-and the fields they include:
+By default `loki.source.cloudflare` fetches logs with the `default` set of fields + the fields passed through `additional_fields`. 
+Here are the different sets of `fields_type` available for selection, and the fields they include:
```

Comment:
I think below we should say something like:

```
* default includes: 
<list of fields> 
plus any extra fields provided via `additional_fields` argument.
```

And then revert line 45 to what it was before. 

Otherwise I get one version of what default means in line 45 and another in line 50.

Reply:
Also we shouldn't use `+` unless it's a mathematical operator.